index.rst

```diff
@@ -6,65 +6,110 @@
+
+The thermal fuses are set to turn off the power at 85C and the heaters will turn on when the cell reaches 5C, warming up the cell to 26C. The temperatures for the heaters are adjustable within the electronics cabinet. In order to change the temperature setting for the thermal fuse, a new fuse will have to be purchased and installed manually.
+
+.. note:: The heaters and thermal sensor to control the heaters are attached directly to the water cell. If the system needs to be disassembled for any reason, please take care when removing these items. 
 
 The white light source can either be manually or remotely controlled. It is important to toggle which mode you are using on the back of the white light source. We use a LabJack to control the WLS, receive errors and set the power. Details on the LabJack control of the WLS are found in `tech note 36 <https://tstn-036.lsst.io>`__.
 
 There are fans internal to the WLS which blow air out of a hole at the top. Secured to this hole is an aluminum duct that pushes the air outside of the enclosed area.
 
+    .. figure:: /_static/mounted_exhaust.png
+       :name: mounted_exhaust
+       :target: ../_images/mounted_exhaust.png
+       :alt: mounted_exhaust
+       :scale: 10 %
+
+    .. figure:: /_static/exhaust_tube.png
+       :name: exhaust_tube
+       :target: ../_images/exhaust_tube.png
+       :alt: exhaust_tube
+       :scale: 10 %
+
 There are two cables that come from the WLS: one for commanding the power level and the other to receive status and error messages. 
 
-Documentation on the White Light Source can be found on Docushare `here <https://docushare.lsst.org/docushare/dsweb/View/Collection-5178>`__
+Further documentation on the White Light Source can be found on Docushare `here <https://docushare.lsst.org/docushare/dsweb/View/Collection-5178>`__
 
 Chiller
----------
-The chiller is used to cool the water cell in the white light source. This uses a mixture of ethanol and water (50/50)?? to carry heat from a shell surrounding the main water cell. The ethanol helps the water in these tubes from freezing. It does not actually pump the water in and out of the cell, but rather carries heat from tubes surrounding the water cell. The control is on the water coming back to the chiller so there is likely to be some delay in large temperature swings. The temperature should be set to 25C (TBC).
+-------
+The chiller is used to cool the water cell in the white light source. This uses a mixture of methanol and water (10/90) to carry heat from a shell surrounding the main water cell. The ,ethanol helps the water in these tubes from freezing. The chiller does not actually pump the water in and out of the cell, but rather carries heat from tubes surrounding the water cell. The control is on the water returning to the chiller so there is likely to be some delay for large temperature swings. The temperature should be set to 25C.
```

Comment:
typo in 2nd sentence ",ethonol"
You should also add that the water should be distilled water (which can obtained from the DI plant that supports the coating chamber)


Is Craig tracking down the methanol? We don't want to forget that...

Reply:
send him a message about this. I will follow up with this.

Reply:
It's on his radar. Made a ticket [SITCOM-573 ](https://jira.lsstcorp.org/browse/SITCOM-573)